feature/more/src/main/java/com/epmedu/animeal/more/profile/ProfileScreen.kt

```diff
@@ -39,10 +40,10 @@ internal fun ProfileScreen(navController: NavController) {
                 modifier = Modifier.padding(top = 40.dp, start = 44.dp)
             )
             // TODO: Put profile fields
-            AnimealShortButton(
+            AnimealButton(
                 text = "Edit",
                 onClick = { viewModel.edit() },
-                modifier = Modifier.padding(top = 32.dp, start = 24.dp)
+                modifier = Modifier.padding(top = 32.dp, start = 24.dp).size(width = 208.dp, height = 60.dp)
```

Comment:
Please reformat this line by Ctrl + Alt + L

Reply:
The line cannot be reformatted with **Ctrl + Alt + L**, since it doesn't violate code style rules: Multiple modifiers can be put in one line if they fit line width.